docs/pages/fund-agents-apps/fund-your-app.mdx

```diff
@@ -244,7 +244,7 @@ You'll need your payer wallet's private key when setting up your [XMTP Gateway S
 
 ### Step 2. Register the payer wallet
 
-1. Use your payer wallet to connect to the XMTP Funding Portal → testnet link coming soon.
+1. Use your payer wallet to connect to the XMTP Funding Portal → testnet-staging link coming soon.
```

Comment:
correct? not available yet?

Reply:
@jhaaaa I think we should share it now! https://funding-portal-testnet-staging.vercel.app

With a cavevat along the lines of "This staging deployment is available for testing deposits and withdrawals with mock currency. Staging balances are currently unaffected by usage."